### PR TITLE
[ci] #3889: Add docker-compose files run check into iroha2 CI

### PR DESCRIPTION
**Pull request by BAStos525**
## Description
1. Add community docker-compose files CI check before image pushing
2. Update community docker-compose files

### Linked issue
#3889 

### Benefits
Validate community docker-compose files integrity and serviceability.

### Checklist

- [ ] I've read `CONTRIBUTING.md`
- [ ] I've used the standard signed-off commit format (or will squash just before merging)
- [ ] All applicable CI checks pass (or I promised to make them pass later)
- [ ] (optional) I've written unit tests for the code changes
- [ ] I replied to all comments after code review, marking all implemented changes with thumbs up

### CODE REVIEW

**Comment by 0x009922** [comment]
I will update `iroha_swarm` with health check addition.

**Comment by 0x009922** [comment]
@BAStos525, I've updated `iroha_swarm` and refined health check configuration as I think would be optimal.

File: docker-compose.local.yml [hunk 1]
```diff
@@ -11,8 +11,8 @@ services:
       IROHA_CONFIG: /config/config.json
       IROHA_PUBLIC_KEY: ed01208BA62848CF767D72E7F7F4B9D2D7BA07FEE33760F79ABE5597A51520E292A0CB
       IROHA_PRIVATE_KEY: '{"digest_function":"ed25519","payload":"8f4c15e5d664da3f13778801d23d4e89b76e94c1b94b389544168b6cb894f84f8ba62848cf767d72e7f7f4b9d2d7ba07fee33760f79abe5597a51520e292a0cb"}'
-      TORII_P2P_ADDR: iroha0:1337
-      TORII_API_URL: iroha0:8080
+      TORII_P2P_ADDR: 0.0.0.0:1337
```
Comment (mversic) [inline docker-compose.local.yml:14]: why this change?

Comment (BAStos525) [reply docker-compose.local.yml:14]: It's required to bind local container address to localhost to enable the health check command. Otherwise container always receive a random local docker network address.

**Review by mversic — APPROVED** [review]

**Review by AlexStroke — DISMISSED** [review]

**Review by AlexStroke — APPROVED** [review]
